File: Backend/src/main/java/SCD/Backend/DTOs/LoggedUserDTO.java
package SCD.Backend.DTOs;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class LoggedUserDTO {
    private Integer id;
    private String email;
    private String name;
    private String jwt;
    private String accessToken;
    private String refreshToken;
}

File: UI/DTOs/LoggedUserDTO.cs
using System.ComponentModel.DataAnnotations;

namespace UI.DTOs
{
    public class LoggedUserDTO
    {
        public int? Id { get; set; }
        public string? Email { get; set; }
        public string? Name { get; set; }
        public string? Jwt {  get; set; }

        public static explicit operator LoggedUserDTO(Task<string?> v)
        {
            throw new NotImplementedException();
        }
    }
}
